getting-started.md

```diff
@@ -6,42 +6,22 @@ permalink: /getting-started/
 ---
 
 Flutter currently supports developers on Mac and Linux (64-bit).
-Windows support is planned.
+Windows support is planned but currently incomplete.
 
-## Setting up your environment
+## Dependencies
 
-### Downloading and installing the Dart SDK
+Flutter depends on the following tools being available in your environment. These are commonly already available on Mac and Linux.
 
-To get started, you need to install the Dart SDK.
-Flutter requires Dart SDK 1.12.2 or greater.
-
-To install the [Dart SDK](https://www.dartlang.org/downloads/):
-
-- Mac:
-  - `brew tap dart-lang/dart && brew install dart`, or, if you don't use
-    homebrew, grab the [latest stable channel build](https://www.dartlang.org/downloads/archive/)
-- Linux: See [www.dartlang.org/downloads/linux.html](https://www.dartlang.org/downloads/linux.html)
-- Windows: Stay tuned, Windows support is planned.
-
-### Checking your PATH
-
-Ensure that the `pub` executable is in your `PATH`.
-
-Run `pub --version` on the command line. If that command
-is not found, you will need to add `DIRECTORY_WHERE_DART_SDK_IS_LOCATED/bin`
-to your PATH.
-
-For example:
-
-```
-$ export PATH=<DART_SDK_DOWNLOAD_LOCATION>/bin:$PATH
-```
+* bash, mkdir, rm
+* git
```

Comment:
The first time you type 'git' on mac, it shows a dialog about the command-line developer tools.  Unclear if we need to walk people through this?